mmdet/apis/train.py

```diff
@@ -180,6 +180,8 @@ def train_detector(model,
 
     # fp16 setting
     fp16_cfg = cfg.get('fp16', None)
+    if fp16_cfg is None and cfg.get('device', None) == 'npu':
```

Comment:
Does it mean when using NPU we use FP16 by default?

Reply:
yes, for Ascend910, it is necessary